app/src/main/java/org/matrix/android/sdk/sample/SampleApp.kt

```diff
@@ -46,4 +40,21 @@ class SampleApp : Application() {
             lastSession.startSync(true)
         }
     }
+
+    private lateinit var matrix: Matrix
```

Comment:
looks like this field should be above the `onCreate`

Reply:
Yes, done: https://github.com/matrix-org/matrix-android-sdk2-sample/pull/11/commits/ae3fe48a0d67d2c650c442b400f030a11b0a5763